### PR TITLE
First pass on sprite physics

### PR DESCRIPTION
**Pull request by vlidholt**
- Adds physics shapes
- Adds physics node (a wrapper for the Box2D world)
- Adds contact callbacks

### CODE REVIEW

File: examples/game/test_bed.dart [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  if (rootBundle != null)
+    return rootBundle;
+  return new NetworkAssetBundle(Uri.base);
+}
+
+final AssetBundle _bundle = _initBundle();
+
+ImageMap _images;
+SpriteSheet _spriteSheet;
+TestBedApp _app;
```
Comment (abarth) [inline examples/game/test_bed.dart]: There's no need for this variable.  You can just pass the object to runApp

File: examples/game/test_bed.dart [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  ]);
+
+  String json = await _bundle.loadString('assets/sprites.json');
+  _spriteSheet = new SpriteSheet(_images['assets/sprites.png'], json);
+
+  _app = new TestBedApp();
+  runApp(_app);
+}
+
+class TestBedApp extends App {
```
Comment (abarth) [inline examples/game/test_bed.dart]: There's no need to extend App anymore.  You can just extend StatelessComponent

**Comment by abarth** [comment]
LGTM

**Comment by vlidholt** [comment]
I'll update test_bed.dart in the next pull request

**Pull request merged**